mm2src/coins/z_coin.rs

```diff
@@ -396,7 +396,7 @@ impl ZCoin {
         let spendable_notes = self
             .spendable_notes_ordered()
             .await
-            .map_err(|err| GenTxError::SpendableNotesError(err.to_string()))?;
+            .map_err(|err| err.map(|err| GenTxError::SpendableNotesError(err.to_string())))?;
```

Comment:
A strange way to map the error, why?

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/pull/2398#discussion_r2093216306

this preserves the location tracing for mmerror.

Reply:
I believe we have `mm_err` for this

Reply:
https://github.com/KomodoPlatform/komodo-defi-framework/blob/4366141c8a7a5d6d3abac87df8827f7cae53d11c/mm2src/mm2_err_handle/src/map_mm_error.rs#L28-L38

Reply:
we indeed do 😂
fixed here [f76866a](https://github.com/KomodoPlatform/komodo-defi-framework/pull/2398/commits/f76866a6a2edcc446d39c256cb1a84e3901b9a2d)